examples/knative_postgresql_customvar/README.md

```diff
@@ -0,0 +1,248 @@
+# Binding an Imported Quarkus app deployed as Knative service with an In-cluster Operator Managed PostgreSQL Database
+
+## Introduction
+
+This scenario illustrates binding a Quarkus application deployed as Knative service with an in-cluster operated managed PostgreSQL Database.
+
+## Actions to Perform by Users in 2 Roles
+
+In this example there are 2 roles:
+
+* Cluster Admin - Installs the operators and Serverless plugin to the OpenShift cluster
+* Application Developer - Imports a Node.js application, creates a DB instance, creates a request to bind the application and DB (to connect the DB and the application).
+
+### Cluster Admin
+
+The cluster admin needs to install operators, knative serving and a builder image into the cluster:
+
+* Service Binding Operator
+* Backing Service Operator
+* Serverless plugin
+  * Service Mesh Opearator
+  * Serverless Operator
+  * Serverless UI
+* Quarkus Native S2i Builder Image
+
+A Backing Service Operator that is "bind-able," in other
+words a Backing Service Operator that exposes binding information in secrets, config maps, status, and/or spec
+attributes. The Backing Service Operator may represent a database or other services required by
+applications. We'll use [postgresql-operator](https://github.com/operator-backing-service-samples/postgresql-operator) to
+demonstrate a sample use case.
+
+We can install all by running the following make target:
+
+```shell
+make install-all
+```
+
+or we can install one by one via the following make targets.
```

Comment:
Suggestion - change the text to read:

or we can install one by one via the following six individual make targets.

Reply:
Thx, updated.